game/games/asecondguy_connect/connect.gd

```diff
@@ -46,9 +45,9 @@ func _empty():
 		line.default_color = player_colors[_last_chip.player_id]
 		_grid.add_child(line)
 	if _end_condition == 0:
-		_info_label.start("Draw")
+		_info_label.start("T_ASECONDGUY_CONNECT_END_MESSAGE_DRAW")
```

Comment:
Localization constants should be global/generic, if reasonable. Like in this case I'd just go with `T_RESULT_DRAW`.

Reply:
I disagree. I think we should vote on some guidelines for choosing those names.

Reply:
Fair enough.🙂 I'm definitely not sure it's a good idea to keep your name inside the constant, though. `CONNECT` makes it unique already.